lib/ChangeDueDateDialog/ChangeDueDateDialog.js

```diff
@@ -99,6 +112,14 @@ class ChangeDueDateDialog extends React.Component {
     return null;
   }
 
+  componentDidMount() {
+    const loans = this.loans();
```

Comment:
Please move this logic to a method and call it there, so componentDidMount would be used for other stuff also.

Reply:
@VictorSoroka1  thanks for you comments. They were handled and changes can be checked for now.

Reply:
Thanks @YevheniiMaltsev.